portal/webpack-development.js

```diff
@@ -7,6 +7,29 @@ const configuration = merge(common, {
   mode: "development",
   devtool: "eval-source-map",
   devServer: {
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Methods": "GET, POST, PUT, DELETE, PATCH, OPTIONS",
+      "Access-Control-Allow-Headers": "X-Requested-With, content-type, Authorization"
+    },
+    setupMiddlewares: (middlewares, devServer) => {
+      if (!devServer) {
+          throw new Error('webpack-dev-server is not defined');
+      }
+
+      middlewares.unshift((req, res, next) => {
+          if (req.method === 'OPTIONS') {
+              res.header('Access-Control-Allow-Origin', '*'); // Replace with specific origin if needed
```

Comment:
Can you add an array/string of Access-Control-Allow-Origin so we don't allow everything? It doesnt matter that much because its just for dev, but this way we can communicate why we have this config

Reply:
Updated code, but have a look and see if any further edits needed there.